src/Api/Billing/Public/Controllers/OrganizationController.cs

```diff
@@ -35,6 +36,42 @@ public OrganizationController(
         _logger = logger;
     }
 
+    [HttpGet("subscription")]
+    [ProducesResponseType(typeof(OrganizationSubscriptionDetailsResponseModel), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.NotFound)]
+    public async Task<IActionResult> GetSubscriptionAsync()
```

Comment:
⛏️ Not sure if this is entirely necessary in Public API, but probably worth enforcing `[SelfHosted(NotSelfHostedOnly = true)]`. Other than that, looks great!

Reply:
good call @amorask-bitwarden! just pushed the change, requested your re-review